File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.cc
#include <bits/stdc++.h>
using namespace std;

int cnt[10];

int main(void)
{
  ios::sync_with_stdio(0);
  cin.tie(0);

  int a, b, c;
  cin >> a >> b >> c;

  long long total = a*b*c;
  string total_st = to_string(total);

  for(int i=0; i<total_st.length(); i++) cnt[total_st.at(i) - '0']++;
  for(int count:cnt) cout << count << '\n';

}

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int a = Integer.parseInt(br.readLine());
        int b = Integer.parseInt(br.readLine());
        int c = Integer.parseInt(br.readLine());
        long total = a*b*c;
        String total_st = String.valueOf(total);

        int[] cnt = new int[10];

        for(int i=0; i<total_st.length(); i++) cnt[total_st.charAt(i) - '0']++;
        for(int count:cnt) System.out.println(count);
    }

}